src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/prisoner/Alias.kt

```diff
@@ -7,6 +7,6 @@ import java.time.LocalDate
 data class Alias(
   val firstName: String? = null,
   val lastName: String? = null,
-  val middleNames: List<String>? = emptyList(),
+  val middleNames: String?,
```

Comment:
Does this need a default like the others? Or should we remove the default values from the others?